doc/boot_arguments.md

```diff
@@ -26,3 +26,14 @@ agama.web.ssl=true agama.web.ssl_cert=http://192.168.122.1/mycert.pem agama.web.
 ```
 
 Changing complex options (e.g., collections) is not supported yet.
+
+## Proxy Setup
+
+Agama supports proxy setup using the `proxy=` kernel command line option like 
+`proxy=http:192.168.122.1:3128`.
```

Comment:
Is this supposed to be a URL?

```suggestion
`proxy=http://192.168.122.1:3128`.
```

If this setting is compatible with some standard or other tools, we should mention it here